src/coprocessor/codec/mysql/decimal.rs

```diff
@@ -835,6 +838,42 @@ pub enum RoundMode {
 }
 
 impl Decimal {
+    /// abs the Decimal into a new Decimal.
+    #[inline]
+    pub fn abs(&self) -> Res<Decimal> {
+        let mut res = self.clone();
+        res.negative = false;
+        Res::Ok(res)
+    }
+
+    /// ceil the Decimal into a new Decimal.
+    pub fn ceil(&self) -> Res<Decimal> {
+        let mut target = if self.frac_cnt > 0 && !self.negative {
```

Comment:
because `ceil(-5.5)` should return `-5`, but `round(-5.5, RoundMode::Ceiling)` will return `-6`.